flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -234,17 +232,6 @@ under the License.
 			<version>${aws.sdkv2.version}</version>
 		</dependency>
 
-		<dependency>
-			<groupId>org.apache.httpcomponents</groupId>
-			<artifactId>httpclient</artifactId>
```

Comment:
These should actually stay. This ensures that if a _user_ bundles this this connector in the jar they still get this version of the dependency. In that scenario what we define in the dependency management is completely ignored.